BUILDING.md

```diff
@@ -8,6 +8,21 @@ You can skip to the [setup guide for Kubernetes](QUICKSTART-EKS.md) or the [setu
 
 ### Dependencies
 
+#### Adequate Disk Space
+
+The build process artifacts and resulting images can consume in excess of 4GB in the local directory.
```

Comment:
`System Requirements` might be a better heading, then we can add recommendations for storage + memory + compute.

Systems should have at least 80 GB of storage available. There's quite a lot of overhead from the intermediate Docker layers that are created.